casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala

```diff
@@ -229,12 +228,12 @@ class InterpreterUtilTest
      *         genesis
      */
     for {
-      b1 <- buildBlock[Task](Seq(genesis.blockHash), deploys = b1DeploysWithCost, now = 1)
-      b2 <- buildBlock[Task](Seq(genesis.blockHash), deploys = b2DeploysWithCost, now = 1)
+      b1 <- buildBlock[Task](Seq(genesis.blockHash), deploys = b1DeploysWithCost, now = 100)
+      b2 <- buildBlock[Task](Seq(genesis.blockHash), deploys = b2DeploysWithCost, now = 200)
       b3 <- buildBlock[Task](
              Seq(b1.blockHash, b2.blockHash),
              deploys = b3DeploysWithCost,
-             now = 2
+             now = 300
```

Comment:
@JosDenmark can you confirm increasing these values are valid for this test to pass. This test was failing with _Unable to consume_ error.